src/components/ProductsTable/__tests__/ProductsTable.test.tsx

```diff
@@ -413,16 +413,14 @@ describe('ProductsTable', () => {
         sku: 'RH123',
         quantity: 2,
         productLine: 'consonants',
-        serviceLevel: 'Standard',
-        unitOfMeasure: { name: 'Mock1', quantity: 'unlimited' }
+        serviceLevel: 'Standard'
```

Comment:
Should this have the capacity in it still? I think this should just be a rename

Reply:
> Should this have the capacity in it still? I think this should just be a rename

I believe these just slipped through the cracks when the story of removing Unit of Measure from Products Table was pushed out. Neither UoM or Capacity should be in Products Table.

Reply:
Ack - good call